runtime/doc/lsp.txt

```diff
@@ -61,47 +61,41 @@ options are not restored when the LSP client is stopped or detached.
 - |K| is mapped to |vim.lsp.buf.hover()| unless |'keywordprg'| is customized or
   a custom keymap for `K` exists.
 
+                                                *gr-default*
+                                                *i_CTRL-S*
+Some keymaps are created unconditionally when Nvim starts:
+- *crn* is mapped in Normal mode to |vim.lsp.buf.rename()|
+- *cra* is mapped in Normal mode to |vim.lsp.buf.code_action()|
```

Comment:
```suggestion
- *crr* is mapped in Normal and Visual mode to |vim.lsp.buf.code_action()|
```

(ignore the `crr` parts if the conclusion was to stick with `cra` - I don't mind which)

Reply:
I think the conclusion is to leave `crr` open for now, and use `crn` for rename and `cra` for code actions. @justinmk?

Reply:
I'm in favor of `crr` for this PR. But I'm not going to block on that.

Reply:
Ok misunderstanding on my part, updated.